data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -438,6 +441,34 @@ private IntStream getValidatorSelector(
     return SafeFuture.completedFuture(Optional.of(result));
   }
 
+  public SafeFuture<Optional<StateSyncCommittees>> getStateSyncCommittees(
+      final String stateIdParam, final Optional<UInt64> epoch) {
+    return defaultStateSelectorFactory
+        .defaultStateSelector(stateIdParam)
+        .getState()
+        .thenApply(maybeState -> maybeState.map(state -> getSyncCommitteesFromState(state, epoch)));
+  }
+
+  private StateSyncCommittees getSyncCommitteesFromState(
+      final tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState state,
+      final Optional<UInt64> epochQueryParam) {
+    final UInt64 epoch = epochQueryParam.orElse(spec.computeEpochAtSlot(state.getSlot()));
+    final UInt64 slot = spec.computeStartSlotAtEpoch(epoch);
+
+    final SyncCommittee committee =
+        spec.getSyncCommitteeUtilRequired(slot).getSyncCommittee(state, epoch);
```

Comment:
I suspect we should use `getSyncCommitteeUtil()` here and return an empty result if it's empty (ie prior to the altair fork).

I guess a bad request response is also reasonable, but should be clear in the API spec what the expected behaviour is.